pkg/provider/gitlab/gitlab.go

```diff
@@ -38,6 +39,9 @@ const (
 	errList                                   = "could not verify if the client is valid: %w"
 	errAuth                                   = "client is not allowed to get secrets"
 	errUninitializedGitlabProvider            = "provider gitlab is not initialized"
+	errNameNotDefined                         = "'find.name' is mandatory"
+	errTagsNotImplemented                     = "'find.tags' is not implemented in the Gitlab provider"
```

Comment:
```suggestion
	errTagsNotImplemented                     = "'find.tags' is not currently supported by Gitlab provider"
```

Just a nit here, since it's not a matter of implementing the feature, but rather that the feature itself cannot be implemented (I think the `path` is accurate 😄 )